### PR TITLE
Adds new integration [evercape/hass-resol-KM2]

### PR DESCRIPTION
**Pull request by evercape**
Resol KM2 Controller custom integration

## Checklist

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: https://github.com/evercape/hass-resol-KM2/releases/tag/2023.11.1
Link to successful HACS action (without the `ignore` key): https://github.com/evercape/hass-resol-KM2/actions/workflows/hacs.yaml
Link to successful hassfest action (if integration): https://github.com/evercape/hass-resol-KM2/actions/workflows/validate.yaml

This is my first publish to HACS. I have read all the guidelines, done the validation, brands were updated, created new version after validation - if for whatever reason I have missed something please let me know. Thanks. M

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -315,6 +315,7 @@
   "erikkastelec/hass-WEM-Portal",
   "eseglem/hass-wattbox",
   "evantaur/seiverkot-consumption",
+  "evercape/hass-resol-KM2",
```
Comment (ludeeus) [inline integration:318]: hassfest action is missing in the repository and failing against the latest release.

Comment (evercape) [reply integration:320]: Thanks - I was on vacation and did not update anything, but the HASS action has been activated again. Somehow it was deactivated by GITHUB.

**Comment by evercape** [comment]
@ludeeus Hi Joakim can you please help me out. I am not sure what I must do in order to get the integration published. I have updated validate and hacs action. Thanks a lot. Martin

**Comment by evercape** [comment]
Thanks so much for your help!

**Pull request merged**